template/template.go

```diff
@@ -9,6 +9,9 @@ func Index(url, title, cname string) string {
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="refresh" content="0; url=%[1]s" />
+  <script type="text/javascript">
+    window.location.href = "%[1]s"
```

Comment:
I think we want to use [`Location.replace`](https://developer.mozilla.org/en-US/docs/Web/API/Location/replace) here instead, since just setting `Location.href` will add the current document to session history, which we probably don't want.

```js
window.location.replace("%[1]s")
```

(I haven't had a chance to test this)